Content/Shader/EdgePicker.shader

```diff
@@ -43,7 +43,13 @@ CGPROGRAM
             v2f vert (appdata v)
             {
                 v2f o;
-                o.pos = UnityObjectToClipPos(v.vertex);
+                o.pos = float4(UnityObjectToViewPos(v.vertex.xyz), 1);
+                //Offsetting the edges to avoid z-fighting problems occuring with Unity 20.2
+                o.pos.xy *= lerp(.99, 1, unity_OrthoParams.w);
+                //Moving edges closer when using orthographic camera
+                o.pos.z *= lerp(.99, .95, unity_OrthoParams.w);
+                o.pos = mul(UNITY_MATRIX_P, o.pos);
```

Comment:
Why is z done separately here? Can this use `ProBuilderCG.UnityObjectToClipPosWithOffset` instead?

Reply:
`ProBuilderCG.UnityObjectToClipPosWithOffset` was created for Metal and it does not seem to behave the same.
I moved this method to `ProBuilderCG.UnityObjectToClipPosWithOffsetMeta`l and create a 'new' `ProBuilderCG.UnityObjectToClipPosWithOffset` to handle EdgePicker as well as Face highlight the same way.
I also modified the Z to behave the same for both camera configs (ortho or not)